canteens/bsr.py

```diff
@@ -0,0 +1,108 @@
+import datetime
+import re
+import urllib.request
+
+
+from bs4 import BeautifulSoup
+from celery.utils.log import get_task_logger
+
+from backend.backend import app, cache, cache_date_format, cache_ttl
+from canteens.canteen import VEGGIE, MEAT, get_next_week, get_current_week
+
```

Comment:
Please add some documentation like in https://github.com/ekeih/OmNomNom/blob/master/canteens/cafenero.py. It is used to generate https://docs.omnbot.io/.

It would be even better if you could add some tests: https://github.com/ekeih/OmNomNom/tree/master/tests
But I will will merge the code also without it ;)

Reply:
I just followed the documentation style from this file: https://github.com/ekeih/OmNomNom/blob/master/canteens/personalkantine.py :smile: 
But I accept that there is room for further improvements. I will see what I can do..